capture-sdk/sdk/src/main/java/net/gini/android/capture/GiniCapture.java

```diff
@@ -154,123 +159,139 @@ public static synchronized boolean hasInstance() {
      * Configure and create a new instance using the returned {@link Builder}.
      *
      * @return a new {@link Builder}
-     * @throws IllegalStateException when an instance already exists. Call {@link #cleanup(Context, String, String, String, String, String, Amount)}
+     * @throws IllegalStateException when an instance already exists. Call {@link #cleanup(Context)}
      *                               before trying to create a new instance
      * @deprecated Please use {@link #newInstance(Context)} which allows instance recreation without having to
-     * call {@link #cleanup(Context, String, String, String, String, String, Amount)} first.
+     * call {@link #cleanup(Context)} first.
      */
     @NonNull
     @Deprecated
     public static synchronized Builder newInstance() {
         if (sInstance != null) {
-            throw new IllegalStateException("An instance was already created. "
-                    + "Call GiniCapture.cleanup() before creating a new instance.");
+            throw new IllegalStateException("An instance was already created. " + "Call GiniCapture.cleanup() before creating a new instance.");
         }
         return new Builder();
     }
 
     /**
      * Configure and create a new instance using the returned {@link Builder}.
      *
-     * @return a new {@link Builder}
      * @param context Android context
+     * @return a new {@link Builder}
      */
     @NonNull
     public static synchronized Builder newInstance(final Context context) {
         if (sInstance != null) {
             if (sInstance.mNetworkRequestsManager != null) {
                 sInstance.mNetworkRequestsManager.cleanup();
             }
-            doActualCleanUp(context);
+            cleanup(context);
         }
         return new Builder();
     }
+
     /**
-     * Destroys the {@link GiniCapture} instance and frees up used resources.
+     * Provides transfer summary to Gini.
      *
      * <p>Please provide the required extraction feedback to improve the future extraction accuracy.
```

Comment:
[Request] Rename to transfer summary.
```suggestion
     * <p>Please provide the required transfer summary to improve the future extraction accuracy.
```